code/src/sixsq/nuvla/server/resources/spec/nuvlabox_status_0.cljc

```diff
@@ -139,13 +183,60 @@
 
 
 (s/def ::resources
-  (-> (st/spec (su/only-keys :req-un [::cpu ::ram ::disks]))
+  (-> (st/spec (su/only-keys :req-un [::cpu ::ram ::disks] :opt-un [::net-stats]))
       (assoc :name "resources"
              :json-schema/type "map"
              :json-schema/description "available and consumed resources"
 
              :json-schema/order 33)))
 
+
+(s/def ::operating-system
+  (-> (st/spec ::core/nonblank-string)
+    (assoc :name "operating-system"
+           :json-schema/description "name of the host OS"
+
+           :json-schema/order 37)))
+
+
+(s/def ::architecture
+  (-> (st/spec ::core/nonblank-string)
+    (assoc :name "architecture"
+           :json-schema/description "platform hw architecture"
+
+           :json-schema/order 38)))
+
+
+(s/def ::hostname
+  (-> (st/spec ::core/nonblank-string)
+    (assoc :name "hostname"
+           :json-schema/description "device hostname"
+
+           :json-schema/order 39)))
+
+
+(s/def ::ip
+  (-> (st/spec ::core/nonblank-string)
+    (assoc :name "ip"
+           :json-schema/description "device IP, as used by the NuvlaBox"
+
+           :json-schema/order 40)))
+
+
+(s/def ::docker-server-version
+  (-> (st/spec ::core/nonblank-string)
+    (assoc :name "docker server version"
+           :json-schema/description "docker server version on the host"
+
+           :json-schema/order 41)))
+
+
+(s/def ::last-boot
+  (-> (st/spec ::core/nonblank-string)
```

Comment:
Why not a timestamp type as `created` attribute to keep it consistent?

Reply:
can be, but need to change the nb agent to make it like that...let me know

Reply:
Please do it, it's better now than later on. Thanks